apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -59,6 +91,27 @@ class ReviewTab extends Component {
           isReadyForReview: false
         });
       });
+
+    if (
```

Comment:
I have a PR open to prevent server calls when we don't have a channelId, so we'll need to add that condition here. Can do that once once of us has merged and we resolve the merge conflict:

https://github.com/code-dot-org/code-dot-org/pull/41931

Reply:
Yeah I think in general it'll be nice to refactor these service calls a bit so we can put in whatever checks we need to.